src/main/java/by/intexsoft/ignatieva/Application.java

```diff
@@ -0,0 +1,10 @@
+package by.intexsoft.ignatieva;
+
+public class Application {
+    public static void main(String args[]) {
+        int[][] matrixSpiral = Spiral.fillInMatrixSpiral(6);
+        int[][] matrixPyramid = Pyramid.fillInMatrixPyramid(6);
```

Comment:
![image](https://user-images.githubusercontent.com/14859301/100109344-cc19d700-2e7c-11eb-8859-12958d398388.png)
please check on odd inputs

Reply:
it's also actual for fillInMatrixSpiral